src/components/Cat.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from "react"
+import {connect} from "react-redux"
+import {removeCat} from "./../action_creators"
+
+function Cat ({src, fact, id, dispatch}) { // todo fix lint: react/jsx-no-bind
+  return (<div className="cat" onClick={() => dispatch(removeCat(id))}>
+    <img className="image" src={src} />
+    <p className="fact">{fact}</p>
+  </div>)
+}
+
+Cat.propTypes = {
+  fact: PropTypes.string.isRequired,
+  id: PropTypes.number.isRequired,
+  src: PropTypes.string.isRequired
+}
+
+function mapStateToProps (state, ownProps) {
```

Comment:
shouldn't need this here - you're unnecessarily creating a higher order component on Cat even though all of its props are passed down from App already